### PR TITLE
Updated readme with getting started steps

### PR DESCRIPTION
**Pull request by aruniverse**
Updated readme with getting started steps. Copied from imjs repo, and removed what I thought to be irrelevant like extract-api and changes to that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,51 @@ Please refer to the [section](CONTRIBUTING.md#adding-a-new-project) in the contr
...
+4. Repeat steps 1-3 until ready to push changes
+5. Add changelog entry (which could potentially cover several commits): `rush change`
+6. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented.
+7. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
+8. Commit the changelog JSON files.
+9. Publish changes on the branch and open a pull request.
+
+If using the command line, steps 5 through 7 above can be completed in one step by running `rushchange.bat` from the imodeljs-core root directory.
+
+> Note: The CI build will break if changes are pushed without running `rush change` and `rush extract-api` (if the API was changed). The fix will be to complete steps 5 through 10.
```
Comment (calebmshafer) [inline README.md:54]: No need for `rush extract-api` reference

File: README.md [hunk 2]
```diff
@@ -20,3 +20,51 @@ Please refer to the [section](CONTRIBUTING.md#adding-a-new-project) in the contr
...
+The version numbers of internal dependencies should not be manually edited.
+These will be automatically updated by the overall _version bump_ workflow.
+Note that the packages are published by CI builds only.
+
+## Updating dependencies/devDependencies on packages external to monorepo
+
+Use these instructions to update dependencies and devDependencies on external packages (ones that live outside of this monorepo).
+
+1. Edit the appropriate `package.json` file to update the semantic version range
+2. Run `rush check` to make sure that you are specifying consistent versions across the repository
```
Comment (calebmshafer) [inline README.md:69]: This step isn't valid in this repo, we don't enforce consistent versioning.

File: README.md [hunk 3]
```diff
@@ -20,3 +20,51 @@ Please refer to the [section](CONTRIBUTING.md#adding-a-new-project) in the contr
...
+2. Ensure unit tests pass when run locally: `rush test`
+3. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
+4. Repeat steps 1-3 until ready to push changes
+5. Add changelog entry (which could potentially cover several commits): `rush change`
+6. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented.
+7. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
+8. Commit the changelog JSON files.
+9. Publish changes on the branch and open a pull request.
+
+If using the command line, steps 5 through 7 above can be completed in one step by running `rushchange.bat` from the imodeljs-core root directory.
```
Comment (calebmshafer) [inline README.md:52]: No need for this whole sentence

File: README.md [hunk 4]
```diff
@@ -20,3 +20,48 @@ Please refer to the [section](CONTRIBUTING.md#adding-a-new-project) in the contr
...
+4. Repeat steps 1-3 until ready to push changes
+5. Add changelog entry (which could potentially cover several commits): `rush change`
+6. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented.
+7. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
+8. Commit the changelog JSON files.
+9. Publish changes on the branch and open a pull request.
+
+> Note: The CI build will break if changes are pushed without running `rush change`. The fix will be to complete steps 5 through 9.
+
+Here is a sample [changelog](https://github.com/Microsoft/web-build-tools/blob/master/apps/rush/CHANGELOG.md) to demonstrate the level of detail expected.
```
Comment (calebmshafer) [inline README.md:54]: Can you update this link?  It's currently broken in the iModel.js repo too.

Comment (calebmshafer) [reply README.md:54]: Updated link is here, https://github.com/microsoft/rushstack/blob/master/apps/rush/CHANGELOG.md

Comment (aruniverse) [reply README.md]: nice catch

**Pull request merged**